rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,424 @@
+    uid=globals()['rdkit_wg_dict'][b._model_id]
+    globals()['view_'+uid].zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+    
+def ShowConformers3D(Mol, protein = None,
+                     useDrawAs = False, drawAs=None, 
+                     propPanel = False, 
+                     colorPanel = False, 
+                     labelPanel = False):
+    
+    """This function initiates required widgets and 3Dmol.js viewer"""
+    
+    # Common suffix for widgets
+    uid=str(time.time()).replace('.','')
+    
+    # Required global objects
+    globals()['rdkit_mol_select_'+uid] = list()
+    globals()['rdkit_conf_select_'+uid] = list()
+    
+    if 'rdkit_wg_dict' not in globals():
+        globals()['rdkit_wg_dict'] = dict()
+    
+    
+    # Right hand panel (widgets)
+    wgListBox=list()
+    itemLayout=Layout(display='flex', flex_flow='row', justify_content='space-between')
+    
+    
+    # To do : Add protein
+    if protein is not None:
+        #print 'adding protein'
+        globals()['protein_'+uid] = protein
+    
+    
+    # molecules and conformers
+    globals()['moldict_'+uid] = ProcessMolContainingObj(Mol)
+    keys=list(globals()['moldict_'+uid].keys())
+    
+    globals()['selected_mols_view_'+uid] = HTML(description='', value=keys[0])
+    globals()['selected_confs_view_'+uid] = HTML(description='', value='0')
+    globals()['molTupleId_'+uid] = Dropdown(description='', options=keys,value=keys[0])
+    globals()['selectMultiMols_'+uid] = Checkbox(description='selectMultiMols', value=False)
+    globals()['selectAllMols_'+uid] = Checkbox(description='selectAllMols', value=False)
+    globals()['confId_'+uid] = Dropdown(description='', options=range(9), value=0)
+    globals()['selectMultiConfs_'+uid] = Checkbox(description='selectMultiConfs', value=False)
+    globals()['selectAllConfs_'+uid] = Checkbox(description='selectAllConfs', value=False)
+    
+    wgListBox.append(Box([Label(value='Mols'),globals()['selected_mols_view_'+uid]], layout=itemLayout))
+    wgListBox.append(Box([Label(value='Confs'),globals()['selected_confs_view_'+uid]], layout=itemLayout))
+    
+    wgListBox.append(Box([Label(value='Mol'),globals()['molTupleId_'+uid]], layout=itemLayout))
+    
+    cbMolSelect=Box([Label(value=''),
+                     HBox([globals()['selectMultiMols_'+uid], 
+                           globals()['selectAllMols_'+uid]])
+                    ], layout=itemLayout)
+    
+    wgListBox.append(cbMolSelect)
+    
+    wgListBox.append(Box([Label(value='confId'),globals()['confId_'+uid]], layout=itemLayout))
+    
+    cbConfSelect=Box([Label(value=''),
+                      HBox([globals()['selectMultiConfs_'+uid], 
+                            globals()['selectAllConfs_'+uid]])
+                     ], layout=itemLayout)
+    
+    wgListBox.append(cbConfSelect)
+    
+    globals()['rdkit_wg_dict'].update({globals()['molTupleId_'+uid]._model_id:uid})
```

Comment:
This is an excellent idea. I have implemented this throughout the whole function. Code looks cleaner.